corehq/ex-submodules/couchexport/shortcuts.py

```diff
@@ -44,7 +44,7 @@ def export_data_shared(export_tag, format=None, filename=None,
 def export_response(file, format, filename, checkpoint=None):
     """
     Get an http response for an export
-    file can be either a StringIO
+    file can be either a io.BytesIO or io.StringIO
     or an open file object (which this function is responsible for closing)
```

Comment:
Thanks for updating the comments as well.

This was the case before, but this is kind of a weirdly spaced comment. If I was skimming through it I'd probably think it could only be either stringio or bytesio and miss the open file obect on the next line